jira-source.kamelet.yaml

```diff
@@ -42,14 +44,15 @@ spec:
         description: A query to filter issues
         type: string
         example: project=MyProject
-      
+
   types:
     out:
       mediaType: application/json
   dependencies:
   - "camel:jackson"
   - "camel:kamelet"
   - "camel:jira"
+  - "mvn:com.fasterxml.jackson.datatype:jackson-datatype-joda:2.12.5"
```

Comment:
Please, double check if we have any and use directly some productized dependency.